src/client/src/utils/cliTemplatesParser.ts

```diff
@@ -12,14 +12,15 @@ export const getFrameworksOptions = (json: any[], type: FRAMEWORK_TYPE): IOption
     longDescription: val.longDescription,
     position: val.position,
     selected: val.selected,
-    svgUrl: "",
+    svgUrl: val.svgUrl, //maybe remove? as it´s always set to ""... or we leave that for the future... I vote to remove this
```

Comment:
do we want to keep this here for the future? Or remove as it was being passed always as empty?

Reply:
talked to Sibille and decided to remove that as we will always use the base64 image from the templates